src/Compilers/Core/Portable/Compilation/Expression.cs

```diff
@@ -46,7 +46,7 @@ public static ConstantValue SynthesizeNumeric(ITypeSymbol type, int value)
                 return SynthesizeNumeric(((INamedTypeSymbol)type).EnumUnderlyingType, value);
             }
 
-            return null;
+            return ConstantValue.Bad;
```

Comment:
I think that creates a constant value that is a 32-bit signed integer with the value 1.